ci-operator/step-registry/openshift/e2e/cert-rotation/prepare-for-shutdown/openshift-e2e-cert-rotation-prepare-for-shutdown-commands.sh

```diff
@@ -121,7 +121,9 @@ function wait-for-valid-lb-ext-kubeconfig {
 cat << 'EOZ' > /tmp/wait-for-kubeapiserver-to-start-progressing.sh
   echo "Waiting for kube-apiserver to start progressing to avoid stale operator statuses"
   export KUBECONFIG=/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/lb-ext.kubeconfig
-  oc --request-timeout=5s wait --for=condition=Progressing clusteroperator/kube-apiserver --timeout=300s 
+  # TODO: kube-apiserver never starts progressing in 4.19+
+  # oc --request-timeout=5s wait --for=condition=Progressing=True clusteroperator/kube-apiserver --timeout=300s
```

Comment:
Is this a bug? In QE CI 4.19 tests, we don't see this.

Reply:
This seems to happen only when instance is being restarted after shutdown/suspend. I'd like to investigate this more later and land the fix to unblock testing now

Reply:
Okay. 
/lgtm